exonum-light-client/src/main/java/com/exonum/client/ExplorerApiHelper.java

```diff
@@ -139,17 +146,15 @@ private static TransactionResult getTransactionResult(
    */
   @Value
   static class SubmitTxRequest {
-    @SerializedName("tx_body")
-    TransactionMessage body;
+    TransactionMessage txBody;
```

Comment:
it will not work because used for serialization `com.exonum.binding.common.serialization.json.JsonSerializer#json` instance is not configured for names with an underscore

Reply:
It will — see the patch to this very file configuring Gson.